src/ua-tool.c

```diff
@@ -73,21 +91,117 @@ static void ua_disable_cb(GObject *object, GAsyncResult *result,
   }
 }
 
+// Called when token configuration file if filled.
+static void ua_attach_call_client(GObject *object, GAsyncResult *result,
+                                  gpointer user_data) {
+  GOutputStream *output_stream = G_OUTPUT_STREAM(object);
```

Comment:
The existing coding style has all the variable definitions at the point of usage rather than at the top of the function.

Reply:
Fixed in most places, I still kept things such error as the beginning since it seems clearer where used.